object-cache.php

```diff
@@ -295,6 +330,12 @@ function delete($id, $group = 'default') {
 	}
 
 	function flush() {
+
+		// Return true is flushing is disabled
+		if ( ! WP_MEMCACHE_FLUSHING ) {
```

Comment:
There are a number of reasons. The main one for me, we used to share our memcache servers with other applications / sites (in our testing environments). Flushing the cache, flushed for everything. Needed a way disable it so we didn't break other apps.